content/docs/ref/pr.md

```diff
@@ -45,7 +45,9 @@ jobs:
       - name: Generate data
         run: echo "Hello World" > output.txt
       - name: Create and merge PR
-        run: gh pr merge --rebase $(cml pr "output.txt")
+        run: |
+          cml ci
+          gh pr merge --rebase $(cml pr "output.txt")
```

Comment:
Not sure if this is required. Should we have `cml ci` auto-run by the `setup-cml` action instead?

Reply:
Sounds like an awesome idea! Maybe we can add a default–false or even default–true option to enable/disable this feature.

Reply:
à la https://github.com/iterative/cml/pull/903 :)

Reply:
https://github.com/iterative/setup-cml/issues/58 :)